client/templates/profilelink/profilelink.js

```diff
@@ -0,0 +1,18 @@
+if (Meteor.isClient) {
+    Template.profilelink.helpers({
+        signedInAs: function() {
+            var user = Meteor.user();
+            if (user) {
```

Comment:
We could. I just abstracted the whole profile link into its own template. We could move this logic elsewhere but I vote we just do that when the use case arises